.github/workflows/docker.yaml

```diff
@@ -0,0 +1,21 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - $default-branch
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build image
+        run: docker-compose -f docker-compose.prod.yaml build
```

Comment:
This docker-compose file was deleted.

Reply:
Disabled workflow, can recreate it with DiiD later for CI image building.